aten/src/ATen/native/transformers/cuda/flash_attn/flash_api.h

```diff
@@ -18,6 +19,7 @@ mha_fwd(const at::Tensor &q,         // batch_size x seqlen_q x num_heads x head
         bool is_causal,
         int window_size_left,
         int window_size_right,
+        const float softcap,
```

Comment:
Why is this change happening at the same time? (Also, nice use of `const`!)

Reply:
So this PR also brings us to a much newer version of flash that has since added these params, I could probably make a stack that adds this dummy arg if helpful for review